src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorHelpers.cs

```diff
@@ -1,14 +1,30 @@
-﻿using System;
-using System.Collections.Generic;
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// MUX Reference ColorHelpers.cpp, tag winui3/release/1.4.2
+
+using System;
 using System.IO;
+using System.Linq;
 using System.Runtime.InteropServices.WindowsRuntime;
 using System.Threading.Tasks;
-using Uno.UI.Helpers.WinUI;
 using Windows.Foundation;
+using Windows.Storage.Streams;
+using Windows.System;
 using Windows.System.Threading;
 using Windows.UI;
-using Windows.UI.Core;
+using Windows.UI.Xaml.Media;
 using Windows.UI.Xaml.Media.Imaging;
+using Uno.Extensions;
+using Uno.UI.Dispatching;
+using Uno.UI.Helpers.WinUI;
+using size_t = System.UInt64; // Uno Doc: size_t is 8 bytes on mosts modern machines
```

Comment:
Defining and using a new `size_t` here IMO is questionable. When converting from C++ to C# this no longer makes sense and just hurts readability of the code. I had converted all cases to use regular C# types and it was working fine.

Reply:
Yeah, I found it questionable as well, but I guess it makes the code nicer to compare. No strong opinions here, could remove if it was found really ugly.

Reply:
Let's let someone else break the tie then. As stated, I usually would default to C#/.NET convention in these cases (and did so in the original port).